app/components/dex-create.jsx

```diff
@@ -135,6 +149,7 @@ function mapStateToProps ({ session }) {
 function mapDispatchToProps (dispatch) {
```

Comment:
cant comment on the line, but add `games` to the args and the return value of `mapStateToProps`